apps/emqx_plugins/src/proto/emqx_plugins_proto_v1.erl

```diff
@@ -0,0 +1,37 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022-2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_plugins_proto_v1).
+
+-behaviour(emqx_bpapi).
+
+-export([
+    introduced_in/0,
+    get_tar/2
+]).
+
+-include_lib("emqx/include/bpapi.hrl").
+
+-define(TIMEOUT, 25000).
+
+-type name_vsn() :: binary() | string().
+
+introduced_in() ->
+    "5.0.21".
+
+-spec get_tar(node(), name_vsn()) -> {ok, binary()} | {error, any}.
+get_tar(Node, NameVsn) ->
+    rpc:call(Node, emqx_plugins, get_tar, [NameVsn], ?TIMEOUT).
```

Comment:
It would be nice to test this RPC with a very large file and with EMQX under traffic. OTP team did some optimizations to make sure large messages don't block up the distribution socket, but there still could be some performance pitfalls related to transfering very large binaries over RPC.

Reply:
I agree, but this is essentially not different to what we already have in `emqx_mgmt_api_plugins.erl`:  
https://github.com/emqx/emqx/blob/master/apps/emqx_management/src/emqx_mgmt_api_plugins.erl#L324
Perhaps we should add validation for dashboard/CLI plugin uploads to check the max size of initially added plugin?

Reply:
I see. If the existing code is already designed like that (I haven't studied it prior to the review), then this PR follows the design, however it is.